metron-deployment/roles/metron_streaming/tasks/main.yml

```diff
@@ -51,4 +51,8 @@
 
 - include: hdfs_purge.yml
 
+- include: create_index_templates.yml
+  when: install_elasticsearch == True
```

Comment:
Yeah, I think I'll just pull it out to a separate roll and figure it out from there.